explore/src/main/scala/explore/config/AdvancedConfigurationPanel.scala

```diff
@@ -76,7 +88,7 @@ sealed trait AdvancedConfigurationPanel[T <: ScienceModeAdvanced, S <: ScienceMo
 
 sealed abstract class AdvancedConfigurationPanelBuilder[
   T <: ScienceModeAdvanced,
-  S <: ScienceModeBasic,
+  S <: ScienceModeBasic: Eq,
```

Comment:
Do we use the `Eq` besides deriving `Reusability`? If not, can we require the `Reusability` directly? It may already be defined.

Reply:
We only use it for Reusability. Reusability is already defined on `ScienceModeAdvanced`, but not on all if it's subtypes. I guess I could define it for the subtypes, and then summon it here.